### ISSUE TITLE
iOS/Android: once permissions are denied camera will flash when the user tries to scan

### ISSUE THREAD

**Issue by natalieeeflores** [mintware-de/flutter_barcode_reader#189]
The app will briefly open the camera when a user denies camera permissions. I am able to reproduce this behavior with the example app in this repo.  

Steps to reproduce:
1. Press `scan` button
2. Deny camera permissions
3. Press `scan` button again

barcode_scan version: 2.0.1

Devices reproduced on: 
iPhone 10R, iOS 13.3.1
Samsung Galaxy S9, Android 10 

![RPReplay_Final1583356036](https://user-images.githubusercontent.com/19349394/75923282-9843e080-5e19-11ea-955d-9acc0ecc2b50.gif)

**Comment by devtronic** [comment]
Please check if the fix in PR #186 works for you🙂

**Comment by natalieeeflores** [comment]
I've been having some issues testing the fix on android. I keep getting build failures when I try to run my project: `Execution failed for task ':barcode_scan:compileDebugKotlin'.`
However, I have been able to run the project on an iOS device and can confirm that the bug is still reproducible

**Comment by devtronic** [comment]
Please run `flutter clean` and then open the Android Module directly (Android Studio -> Right click on your project root -> Flutter -> Open Android Module in Android Studio)

**Comment by natalieeeflores** [comment]
Unfortunately, my project still isn't running on android after following those steps. Here's the output from android studio: 
`/.pub-cache/git/flutter_barcode_reader-ea67104d0024de29a73602a8c35c279900db20b7/android/src/main/kotlin/de/mintware/barcode_scan/BarcodeScannerActivity.kt: (33, 30): Unresolved reference: it`

I'm running it on a new project that I've created, I followed the instructions in the readme, and copy-pasted the code in example/lib/main.dart. Thanks for your help in all this!
